src/h2load.cc

```diff
@@ -1493,6 +1647,7 @@ int main(int argc, char **argv) {
     shared_nva.emplace_back(":authority", config.host);
   }
   shared_nva.emplace_back(":method", config.data_fd == -1 ? "GET" : "POST");
+  shared_nva.emplace_back("user-agent", "h2load nghttp2/" NGHTTP2_VERSION);
```

Comment:
I agree with you for the use case, especially for the UA string analysis on public server.  nghttp2.org sometimes gets lots of requests seemingly generated by h2load.

Theoretically, adding header will increase server load (HPACK huffman decoding), so it could reduce some numbers.  But I don't think it is not much an issue, since we can re-run h2load as needed.